rpm/SPECS/contextBroker.spec

```diff
@@ -31,12 +31,12 @@ Release:   %{broker_release}
 License:   AGPLv3
 Group:     Applications/Engineering
 Vendor:     Telefónica I+D
-Packager:   Fermín Galán <fermin@tid.es>
+Packager:   Fermín Galán <fermin.galanmarquez@telefonica.com>
 URL:        http://catalogue.fiware.org/enablers/publishsubscribe-context-broker-orion-context-broker
 Source:     %{name}-%{broker_version}.tar.gz
 BuildRoot: /var/tmp/%{name}-buildroot
-Requires:  libstdc++, boost-thread, boost-filesystem, libmicrohttpd, libcurl, logrotate
-Buildrequires: gcc, cmake, gcc-c++, libmicrohttpd-devel, libcurl-devel, boost-devel
+Requires:  libstdc++, boost-thread, boost-filesystem, gnutls, libcurl, logrotate
+Buildrequires: gcc, cmake, gcc-c++, gnutls-devel, libcurl-devel, boost-devel
```

Comment:
Removing libmicrohttpd as dependency has introduced gnutl, maybe due to a transient depedency previously hiden by libmicrohttpd. Maybe they are other dependencies hiden in that way... let's have a look to the dependencies of libmicrohttpd package itself to know.